pkg/debugapi/peer.go

```diff
@@ -35,6 +35,14 @@ func (s *server) peerConnectHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	s.Addressbook.Put(address, addr)
```

Comment:
didn't way say that the topology driver will actually insert the entries to the address book?

Reply:
It's explained a bit in the desc of the PR. You are correct, we did say this, but when I was implementing I remembered that we also said on some meetings that we would like to avoid the mutliaddr in the interface of the topology and hive, so addressbook is here only as a collection of overlay->multiaddr pairs. Does this makes sense to you? @janos as well. If not, then we don't actually even need addressbook in the full connectivity implementation, but interface would know about mutliaddr, and also hive handler would have it as well. LTMWYT

Reply:
Yes, I would like to say it again, topology driver should not know about underlay address in any of its form. Topology driver should be responsible only of overlay topology.

Reply:
cool :+1: